ospatch/updates.go

```diff
@@ -30,22 +30,22 @@ const (
 	rpmquery = "/usr/bin/rpmquery"
 )
 
-func getBtime(stat string) (int, error) {
+func getBtime(stat string) (int64, error) {
 	f, err := os.Open(stat)
 	if err != nil {
 		return 0, fmt.Errorf("error opening %s: %v", stat, err)
 	}
 	defer f.Close()
 
-	var btime int
+	var btime int64
 	scnr := bufio.NewScanner(f)
 	for scnr.Scan() {
 		if bytes.HasPrefix(scnr.Bytes(), []byte("btime")) {
 			split := bytes.SplitN(scnr.Bytes(), []byte(" "), 2)
 			if len(split) != 2 {
 				return 0, fmt.Errorf("error parsing btime from %s: %q", stat, scnr.Text())
 			}
-			btime, err = strconv.Atoi(string(bytes.TrimSpace(split[1])))
+			btime, err = strconv.ParseInt(string(bytes.TrimSpace(split[1])), 10, 0)
```

Comment:
Should the bitSize be `64`? Otherwise, this is identical to [Atoi](https://golang.org/pkg/strconv/#Atoi)

> Atoi is equivalent to ParseInt(s, 10, 0), converted to type int.